salt/orchestrate/edx/init.sls

```diff
@@ -1,6 +1,11 @@
 {% from "orchestrate/aws_env_macro.jinja" import VPC_NAME, VPC_RESOURCE_SUFFIX,
  ENVIRONMENT, BUSINESS_UNIT, PURPOSE_PREFIX, subnet_ids with context %}
 
+ {% set env_settings = salt.pillar.get('environment_settings') %}
+ {% set bucket_prefixes = |
```

Comment:
The pipe character is only for multiline YAML strings. You can remove it here and bring the next line up to the same line.